cryptocurrency/cryptoApp.app

```diff
@@ -0,0 +1,6 @@
+type=script
+scriptType=python
+enabled=true
+id=crypto
```

Comment:
Typically, we'll want to use a namespaced/unique id. It's *ok* for quick examples to have a simple id. But if we think there's any potential overlap for other examples we may create, it makes sense to have more unique from the start. `io.deephaven.examples.crypto.coingecko`.